dietpi/dietpi-software

```diff
@@ -7165,12 +7168,13 @@ exec sudo -u $ha_user dash -c '$ha_pyenv_activation; exec pip3 install -U homeas
 			Banner_Configuration
 
 			# Openbox + PCmanFM + panel configs
-			mkdir -p /root/.config/{openbox,pcmanfm/LXDE,lxpanel/LXDE/panels}
-			G_THREAD_START curl -sSfL https://raw.githubusercontent.com/$G_GITOWNER/DietPi/$G_GITBRANCH/.conf/desktop/lxde/lxde-rc.xml -o /root/.config/openbox/lxde-rc.xml
-			G_THREAD_START curl -sSfL https://raw.githubusercontent.com/$G_GITOWNER/DietPi/$G_GITBRANCH/.conf/desktop/lxde/pcmanfm.conf -o /root/.config/pcmanfm/LXDE/pcmanfm.conf
-			G_THREAD_START curl -sSfL https://raw.githubusercontent.com/$G_GITOWNER/DietPi/$G_GITBRANCH/.conf/desktop/lxde/pcmanfm-desktopitems.conf -o /root/.config/pcmanfm/LXDE/desktop-items-0.conf
-			G_THREAD_START curl -sSfL https://raw.githubusercontent.com/$G_GITOWNER/DietPi/$G_GITBRANCH/.conf/desktop/lxde/panel -o /root/.config/lxpanel/LXDE/panels/panel
-
```

Comment:
Great to have these applied for all users. Since some of those files are part of the DEB packages themselves, we need to assure that those are marked as conffiles, so that an APT upgrade won't revert our overwrite. But otherwise I'm okay with it. Same for LXQt. I'll do a test install to verify this.

Another thing we should do is comparing our configurations with the ones from the DEB packages/defaults, since those haven't been updated for a while. Probably we can or need to do some update or we could replace a complete own config file with a few single settings changes. E.g. on LXDE I know that the panel background is an ugly image file which should be replaced with a simple grey background colour. As well the panel height is slightly increased in our config, but otherwise I'm not sure if any other differences are worth to maintain a complete own config.

Reply:
I will leave the actual config files out of this pr for now. I do not think, that I have enough experience with WM-config-files. Also that would be a whole other rabbit-hole, that would be opened.

Reply:
Upon rereading this pr, I had the Idea, that this should be an issue?